pkg/gui/controllers/files_controller.go

```diff
@@ -748,6 +755,90 @@ func (self *FilesController) createStashMenu() error {
 	})
 }
 
+func (self *FilesController) openCopyMenu() error {
+	copyNameNoItemSelectionErr := utils.ResolvePlaceholderString(
+		self.c.Tr.FileCopyNoItemSelectedError,
+		map[string]string{"target": "name"},
+	)
```

Comment:
This doesn't work; you are putting a non-translated string into a placeholder of a translated string.

If you are thinking about solving this by making "name" a translated string too, please don't. Stitching together translated strings like this is not good practice; in other languages the grammar might be sufficiently different to make this hard to translate. Also, it's not obvious to translators how the strings are used. If you must have different texts for the different cases, you should duplicate the whole strings. However, in this case I don't think it's worth it, I'd just try to come up with a generic wording that works for all cases.

Reply:
You're right; I was focusing on the code and didn't think about this. I'll put in there a generic string - I think it's not worth duplicating the strings. Thanks for pointing that out.